### DISCUSSION TITLE
Docs: lualibs directive

### DISCUSSION THREAD

**Discussion by dgcampea** [jcorporation/myMPD#894] [Q&A] [answered]
myMPD version: N/A

URL: https://jcorporation.github.io/myMPD/configuration/

**Describe the bug**

`lualibs` description points to a dead link <https://jcorporation.github.io/myMPD/references/scripting>.
As a consequence, it's hard to determine the meaning of its default value (`all`).

**Comment by jcorporation** [comment]
Correct link is https://jcorporation.github.io/myMPD/scripting/

**Comment by dgcampea** [comment]
Sorry, after taking a glance at the link it's still unclear to me what the directive actually means. What are the possible values for it?

**Comment by jcorporation** [comment, marked as answer]
This passage should describe it: https://jcorporation.github.io/myMPD/scripting#lua-standard-libraries

I reworked it to make it hopefully better understandable.
